info.json

```diff
@@ -128,6 +113,25 @@
                 "form"
             ]
         },
+        "views": {
+            "navigation": [
```

Comment:
In navigation, we should select only required navigation.
while export click the "eye icon" near navigation and select only required navigation